pkg/acsengine/defaults.go

```diff
@@ -69,10 +69,11 @@ var (
 	}
 
 	//DefaultRHELOSImageConfig is the RHEL Linux distribution.
+	//Being used if no image or distro specified. Default Distro -> RHEL - > this
 	DefaultRHELOSImageConfig = AzureOSImageConfig{
-		ImageOffer:     "RHEL",
-		ImageSku:       "7.3",
-		ImagePublisher: "RedHat",
+		ImageOffer:     "acsengine-preview",
+		ImageSku:       "rhel74",
+		ImagePublisher: "redhat",
 		ImageVersion:   "latest",
```

Comment:
This should change when code goes live and we will know which images will be available for generic public. Now RHEL does not work but acsengine-preview does.